### PR TITLE
Add rubygem-mqtt package

### PR DESCRIPTION
**Pull request by Odilhao**

### CODE REVIEW

**Comment by Odilhao** [comment]
To unblock #7288

File: comps/comps-foreman-plugins-rhel7.xml [hunk 1]
```diff
@@ -114,6 +114,7 @@
       <packagereq type="default">tfm-rubygem-jquery-matchheight-rails</packagereq>
       <packagereq type="default">tfm-rubygem-kubeclient</packagereq>
       <packagereq type="default">tfm-rubygem-logify</packagereq>
+      <packagereq type="default">rubygem-mqtt</packagereq>
```
Comment (evgeni) [inline comps/comps-foreman-plugins-rhel7.xml:117]:
```suggestion
      <packagereq type="default">tfm-rubygem-mqtt</packagereq>
```

File: comps/comps-foreman-plugins-rhel7.xml [hunk 2]
```diff
@@ -272,6 +273,7 @@
       <packagereq type="default">tfm-rubygem-jquery-matchheight-rails-doc</packagereq>
       <packagereq type="default">tfm-rubygem-kubeclient-doc</packagereq>
       <packagereq type="default">tfm-rubygem-logify-doc</packagereq>
+      <packagereq type="default">rubygem-mqtt-doc</packagereq>
```
Comment (evgeni) [inline comps/comps-foreman-plugins-rhel7.xml:276]:
```suggestion
      <packagereq type="default">tfm-rubygem-mqtt-doc</packagereq>
```

File: packages/plugins/rubygem-mqtt/rubygem-mqtt.spec [hunk 3]
```diff
@@ -0,0 +1,93 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
+%global gem_name mqtt
+%global plugin_name mqtt
+%global foreman_min_version 3.0
```
Comment (evgeni) [inline packages/plugins/rubygem-mqtt/rubygem-mqtt.spec:7]: which template did you use? it's not a foreman plugin, so I think the `scl` template is the correct one.

**Review by evgeni — CHANGES_REQUESTED** [review]

File: packages/plugins/rubygem-mqtt/rubygem-mqtt.spec (hunk 3 above)
Comment (Odilhao) [reply packages/plugins/rubygem-mqtt/rubygem-mqtt.spec:7]: I used the plugin one, I'll do it again

File: comps/comps-foreman-plugins-rhel7.xml [hunk 4]
```diff
@@ -17,6 +17,7 @@
       <packagereq type="default">ipxe-bootimgs</packagereq>
       <packagereq type="default">puppet-foreman_scap_client</packagereq>
       <packagereq type="default">puppetlabs-stdlib</packagereq>
+      <packagereq type="default">tfm-rubygem-mqtt</packagereq>
```
Comment (evgeni) [inline comps/comps-foreman-plugins-rhel7.xml:20]: dupe of line 118?

File: comps/comps-foreman-plugins-rhel7.xml [hunk 5]
```diff
@@ -180,6 +182,7 @@
         Do not edit this section manually and use ./comps_doc.sh script
       -->
       <!--RGD-START-->
+      <packagereq type="default">tfm-rubygem-mqtt-doc</packagereq>
```
Comment (evgeni) [inline comps/comps-foreman-plugins-rhel7.xml:185]: dupe of line 278?

File: comps/comps-foreman-plugins-rhel7.xml (hunk 4 above)
Comment (Odilhao) [reply comps/comps-foreman-plugins-rhel7.xml]: Removed the dupe

**Review by evgeni — APPROVED** [review]

**Comment by evgeni** [comment]
```console
% koji add-pkg --owner kojiadm foreman-plugins-nightly-el8 rubygem-mqtt                                                                            
Adding 1 packages to tag foreman-plugins-nightly-el8
% koji add-pkg --owner kojiadm foreman-plugins-nightly-rhel7 tfm-rubygem-mqtt
Adding 1 packages to tag foreman-plugins-nightly-rhel7
```

**Comment by adamruzicka** [comment]
3.1 CP https://github.com/theforeman/foreman-packaging/pull/7350

**Pull request merged**